umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/echo10.clj

```diff
@@ -185,16 +184,9 @@
   [price format]
   (util/remove-nil-keys
     {:Fees price
-     :Format format
+     :Format (value-of format ".")
```

Comment:
why are we making this change?

Reply:
The add-data-formats function below is no longer needed since the parsing of the formats has been moved to the indexing service. In that function (value-of ..) on formats was done in the deleted function, so it now needs to be done here.

Reply:
We didn't change the ECHO10 format, I don't see why we are changing this code. Is there a bug here that you fixed?

Reply:
Several weeks ago I parsed out ECHO 10 and DIF 10 format.  data providers but data formats and notes together in the same field. So I parsed it out when we translated it from the specifications into UMM.  I did this so that we could humanize the data formats and also use them as facets. I don't think that was correct and now that we are creating valid values for the field users wanted a FormatDescription field where they could put notes about the format.  The parsing code I moved into the indexing service where I should have put it originally. Now I am removing the parsing code and we are now reading the format values as the data provider put them in.